airlock/views/request.py

```diff
@@ -475,6 +483,39 @@ def file_withdraw(request, request_id, path: str):
     return redirect(redirect_url)
 
 
+@instrument(func_attributes={"release_request": "request_id"})
+@require_http_methods(["POST"])
+def request_multiselect(request, request_id: str):
+    release_request = get_release_request_or_raise(request.user, request_id)
+
+    multiform = MultiselectForm(request.POST)
+
+    if not multiform.is_valid():
+        display_form_errors(request, multiform.errors)
+    else:
+        action = multiform.cleaned_data["action"]
+        if action != "withdraw_files":
+            raise Http404(f"Invalid action {action}")
+
+        errors = []
+        successes = []
+        for path_str in multiform.cleaned_data["selected"]:
+            path = UrlPath(path_str)
+            try:
+                bll.withdraw_file_from_request(release_request, path, request.user)
+                successes.append(f"The file {path} has been withdrawn from the request")
+            except exceptions.RequestPermissionDenied as exc:
+                errors.append(str(exc))
+
+        if errors:
+            display_multiple_messages(request, errors, "error")
+        if successes:
+            display_multiple_messages(request, successes, "success")
```

Comment:
for interest, I was just working near [the equivalent place in workspace.py](https://github.com/opensafely-core/airlock/blob/main/airlock/views/workspace.py#L330-L332), which has a slightly different behaviour - I think I prefer this though.

Reply:
Ah yeah, I hadn't seen that. I think it's better to keep them separate though.

Reply:
once this has merged I think I'll change workspace to match